<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfirmSpectral</class>
 <widget class="QDialog" name="ConfirmSpectral">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>921</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/QRap/Spectral.png</normaloff>:/QRap/Spectral.png</iconset>
  </property>
  <widget class="QWidget" name="gridLayout">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>866</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QGridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Select Reciever Station :</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="comboRx"/>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Frequency Resolution (kHz) : </string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QDoubleSpinBox" name="doubleFreqRes"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayout_2">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>310</y>
     <width>178</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QGridLayout">
    <item row="0" column="1">
     <widget class="QPushButton" name="pushCancel">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="pushOk">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="tableInstallations">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>10</y>
     <width>911</width>
     <height>251</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Site</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Sector</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Radio Inst ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>New Column</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Project</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Technology</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>FlagZ</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Status</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Radius</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources>
  <include location="../qrap.qrc"/>
 </resources>
 <connections/>
</ui>
